package/kernel/linux/modules/hwmon.mk

```diff
@@ -9,6 +9,7 @@ HWMON_MENU:=Hardware Monitoring Support
 
 define KernelPackage/hwmon-core
   SUBMENU:=$(HWMON_MENU)
+  DEPENDS:=+!LINUX_6_6:kmod-i2c-core
```

Comment:
Now this looks really weird, why should hwmon core depend on I2C support?

Reply:
This is likely the culprit https://github.com/torvalds/linux/commit/4ce6e8a859f0503d97aac6869bc3b1a24b15601d.

Reply:
Ahh, good old IS_REACHABLE

Reply:
@namiltd In which scenario did you hit this issue?

Reply:
https://github.com/namiltd/openwrt/pull/60 ramips/mt7620 + lm75

Reply:
Ideally I can try to make it a dedicated module... Strange that it was accepted upstream as it really cause these kind of hassle.